src/main/java/de/muenchen/isi/domain/model/abfrageInBearbeitungSachbearbeitung/BauleitplanverfahrenInBearbeitungSachbearbeitungModel.java

```diff
@@ -13,4 +14,6 @@ public class BauleitplanverfahrenInBearbeitungSachbearbeitungModel extends Abfra
     private List<AbfragevarianteBauleitplanverfahrenSachbearbeitungInBearbeitungSachbearbeitungModel> abfragevarianten;
 
     private List<AbfragevarianteBauleitplanverfahrenInBearbeitungSachbearbeitungModel> abfragevariantenSachbearbeitung;
+
+    private VerortungModel verortung;
```

Comment:
Das Attribut verortung ungefähr dort einordnen wo es sich auch in der Klasse BauleitplanverfahrenModel befindet.

Also hier vor den beiden List-Attributen.